src/pages/Admin/Charity/EditProfile/useEditProfile.ts

```diff
@@ -51,6 +53,17 @@ export default function useEditProfile() {
         return showModal(TxPrompt, { error: "No changes detected" });
       }
 
+      if (update.slug !== initial.slug) {
+        const result = await endowment({ id: update.slug });
```

Comment:
Important: just a note that the same check must be performed in the backend as well (`update-endow-profile`), as it's possible some (malicious) actor sends a direct curl request to update profile to our lambda.

Reply:
you could test that this is already the case, remove this check and submit an existing slug

Reply:
Ah, missed the changes that introduces the slug in the first place, so all is good